ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml

```diff
@@ -337,29 +341,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
-- as: aws-ipi-ovn-fips-p2-f28
+- as: aws-ipi-ovn-fips-efs-p2-f28
   cron: 21 23 26 * *
   steps:
     cluster_profile: aws-qe
+    dependency_overrides:
+      OO_INDEX: quay.io/openshift-qe-optional-operators/aosqe-index:v4.10
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes'
       TAG_VERSION: '@4.10'
     test:
     - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
-- as: aws-ipi-ovn-fips-p3-f28-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips-efs
+- as: aws-ipi-ovn-fips-efs-p3-f28-destructive
```

Comment:
Once the test name get changed, need to re-generated its cron accordingly using https://raw.githubusercontent.com/openshift/release/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh:
```
$ generate-cron-entry.sh aws-ipi-ovn-fips-efs-p3-f28-destructive openshift-openshift-tests-private-release-4.10__amd64-nightly.yaml
cron: 32 5 20 * *
```